Subject: [PATCH] Disable Android build
---
Index: box2d-jni/settings.gradle.kts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/box2d-jni/settings.gradle.kts b/box2d-jni/settings.gradle.kts
--- a/box2d-jni/settings.gradle.kts	(revision 5f4387d62991748b36d647c8e845df1d27743040)
+++ b/box2d-jni/settings.gradle.kts	(date 1764514249779)
@@ -17,7 +17,7 @@
 rootProject.name = "box2d-jni"
 
 include("box2d-jni")
-include("box2d-jni-android")
+//include("box2d-jni-android")
 include("box2d-jni-natives-linux")
 include("box2d-jni-natives-linux-arm64")
 include("box2d-jni-natives-macos")
Index: box2d-jni/build.gradle.kts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/box2d-jni/build.gradle.kts b/box2d-jni/build.gradle.kts
--- a/box2d-jni/build.gradle.kts	(revision 5f4387d62991748b36d647c8e845df1d27743040)
+++ b/box2d-jni/build.gradle.kts	(date 1764514249772)
@@ -1,5 +1,5 @@
 plugins {
-    alias(libs.plugins.androidLibrary) apply false
+    //alias(libs.plugins.androidLibrary) apply false
     alias(libs.plugins.mavenPublish) apply false
 }
 
